components/application-mgt/org.wso2.carbon.device.application.mgt.api/pom.xml

```diff
@@ -225,5 +225,195 @@
             <groupId>javax.ws.rs</groupId>
             <artifactId>javax.ws.rs-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.devicemgt</groupId>
+            <artifactId>org.wso2.carbon.apimgt.annotations</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.sonatype.sisu</groupId>
+            <artifactId>sisu-guice</artifactId>
+            <version>2.1.7</version>
+        </dependency>
+        <dependency>
+            <groupId>org.sonatype.sisu</groupId>
+            <artifactId>sisu-guice</artifactId>
+            <version>2.1.7</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.devicemgt</groupId>
+            <artifactId>org.wso2.carbon.apimgt.annotations</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.devicemgt</groupId>
+            <artifactId>org.wso2.carbon.apimgt.annotations</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.devicemgt</groupId>
+            <artifactId>org.wso2.carbon.apimgt.annotations</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.ode.wso2</groupId>
+            <artifactId>ode</artifactId>
+            <version>1.3.5-wso2v14</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.devicemgt</groupId>
+            <artifactId>org.wso2.carbon.apimgt.annotations</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.ode.wso2</groupId>
+            <artifactId>ode</artifactId>
+            <version>1.3.5-wso2v14</version>
```

Comment:
Please add **all** these dependencies to the <dependencyManagement> in the [root pom](https://github.com/wso2/carbon-device-mgt/blob/eae50efeee424a1d507dcdb88f1a2d97bcae39d6/pom.xml#L58). Version should be added as a property as in [here](https://github.com/wso2/carbon-device-mgt/blob/eae50efeee424a1d507dcdb88f1a2d97bcae39d6/pom.xml#L65).